backend/src/main/kotlin/app/ehrenamtskarte/backend/freinet/webservice/schema/types/FreinetApiAgency.kt

```diff
@@ -0,0 +1,8 @@
+package app.ehrenamtskarte.backend.freinet.webservice.schema.types
+
+data class FreinetApiAgency(
+    val arsList: List<String>,
```

Comment:
:wrench: Can we rename this to something meaningful? What does `ars` stand for?

Reply:
AmtlicherRegionalSchlüssel